test/tests/functional/pbs_hook_timeout.py

```diff
@@ -71,9 +70,12 @@ def setUp(self):
         self.server.manager(MGR_CMD_CREATE, NODE, id=self.hostB)
 
         self.server.manager(MGR_CMD_CREATE, NODE, id=self.hostC)
-
+        for mom in self.moms.values():
+            self.server.expect(NODE, {'state': 'free'}, id=mom.shortname)
```

Comment:
why is this change needed? the setUp() checks if number of moms is exactly 3 or not.

Reply:
but the test will get skipped if the no.of moms is not 3.

Reply:
@prakashcv13: The count in expect function considers the vnode as well, so the count differs with no of moms, whereas self.moms.values just consider the natural(base) nodes.

Reply:
Ok. On a cpuset system, if there is one base node and 3 vnodes, the test will get skipped, is that correct?

Reply:
never mind. I understand what this change does.